src/utils/next.ts

```diff
@@ -74,6 +75,13 @@ export const scaffold = (wrapped : ScaffoldedGetServerSideProps, options? : Scaf
             ctx.z.setTokenData(reqWithSession.session.tokenData);
         }
 
+        try {
+            ctx.user = await ctx.z.resource('users', 'me').get();
```

Comment:
It should be possible to reuse this user object further down in the file, to avoid having to retrieve the user again.